docs-source/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -229,13 +229,13 @@ instance will not ignore sessions and will have a longer timeout.
 
 ### Restarting servers
 
-The operator runtime automatically recreates (restarts) server pods when properties on the domain resource that affect server pods change (such as `image`, `volumes`, and `env`).
-The `restartVersion` property on the domain resource lets you force the operator to restart a set of server pods.
+The operator automatically recreates (restarts) WebLogic Server instance Pods when fields on the Domain that affect Pod generation change (such as `image`, `volumes`, and `env`).
+The `restartVersion` field on the Domain lets you force the operator to restart a set of WebLogic Server instance Pods.
 
-The operator runtime does rolling restarts of clustered servers so that service is maintained.
+The operator does rolling restarts of clustered servers so that service is maintained.
 
-#### Properties that cause servers to be restarted
-The operator will restart servers when any of the follow properties on the domain resource that affect the server are changed:
+#### Fields that cause servers to be restarted
+The operator will restart servers when any of the follow fields on the Domain that affect the WebLogic Server instance Pod generation are changed:
```

Comment:
introspectVersion should be mentioned here to cover the MII case.  E.g. if a MII introspectVersion re-introspect detects a model change (including changes to secrets or config maps that the model depends on), then MII will subsequently roll the domain in order to propagate the model change.

Reply:
of course this isn't applicable to DII or DIPV.